### PR TITLE
Auto deploy docs

### PR DESCRIPTION
**Pull request by tephenavies**
Various improvements to docs + Netlify auto deploy

Automatically deploys to https://auto-docs.terria.io

Build command (due to weird JSDoc problem):
```bash
gulp docs && rm wwwroot/doc/reference/module.html#.exports
```

### CODE REVIEW

**Comment by tephenavies** [comment]
Fixes #4534.

File: doc/deploying/setting-up-a-region-mapping-server.md [hunk 1]
```diff
@@ -1,53 +1,3 @@
...
-        },
-
-* `"SA4"`: this identifier does not serve any machine-readable purpose outside this file.
-* `layerName`: the WMS layer of your new regions, including the workspace.
-* `server`: the URL of your GeoServer, up to and including `/ows`.
-* `regionProp`: the name of the attribute containing region identifiers that will be matched against (case-sensitive)
-* `aliases`: alias of CSV column header names that will be recognised as matching this kind of feature. Must be lowercase.
-* `description`: May be used in GUI elements and error messages.
\ No newline at end of file
+The current method to create a new set of boundaries not currently recognised by TerriaJS is detailed in [this repository](https://github.com/TerriaJS/boundary-tiles). You'll find instructions there to create new region mapping boundaries using a zipped shapefile or GeoJSON file as well as instructions you can follow to host the vector tiles on Amazon Web Services in an S3 bucket + CloudFront, but any static file hosting service will work.
```
Comment (soyarsauce) [inline doc/deploying/setting-up-a-region-mapping-server.md:3]: Great to shift this to that repo

**Review by soyarsauce — APPROVED** [review]

**Comment by tephenavies** [comment]
It worked! Failure this time is due to Travis + Sauce. I can't seem to restart Travis builds any more.

**Comment by soyarsauce** [comment]
So, do we merge this?

**Comment by tephenavies** [comment]
Yes. This is ready to merge. After not letting me restart builds, Travis let me restart them on Saturday while I was closing old tabs. The full list of steps in the docs pipeline is here: https://github.com/TerriaJS/terriajs/issues/4811

**Pull request merged**